docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,187 @@
+# Getting started with supercomputing at CSC
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computing! This page provides guidance for getting started with
+using our HPC resources.
+
+It is recommended that new users complete the CSC Computing Environment course,
+which provides an in-depth introduction to CSC services. The next teaching
+instance can be found in the
+[CSC training calendar](https://csc.fi/en/trainings/training-calendar/).
+A [self-learning implementation](https://csc.fi/en/training-calendar/csc-computing-environment-self-learning/)
+is also available. The
+[practical exercises](https://csc-training.github.io/csc-env-eff/) are
+accessible without having to sign up, and quite useful by themselves.
+
+## Available systems
+
+!!! note "Terms of use"
+    Note that in order to use any of the systems, you must first accept its
+    terms of use in the MyCSC service.
+
+### Puhti
+
+New users are recommended to start working on the
+[Puhti supercomputer](../../computing/available-systems.md#puhti).
+Compared to Mahti, it has much more pre-installed software, more GPU nodes, and
+typically more available memory per CPU. Additionally, GPU nodes and some of the CPU nodes
+on Puhti have [fast local NVMe storage](../../computing/disk.md#temporary-local-disk-areas).
+
+### Mahti
+
+If you know that your computations are highly parallelizable, you should
+consider running them on the
+[Mahti supercomputer](../../computing/available-systems.md#mahti).
+Compared to Puhti, Mahti has many more CPU nodes and cores per node. Mahti is
+intended for computations that are able to effectively utilize at least an
+entire CPU node.
+
+Additionally, while Mahti has fewer GPU nodes than Puhti, the A100 GPUs on Mahti
+are considerably more powerful than the V100 GPUs on Puhti, which makes Mahti
+also suitable for large-scale machine learning applications. In contrast to Puhti,
+only GPU nodes have fast local NVMe storage available on Mahti.
+
+### LUMI
+
+The [LUMI supercomputer](../../computing/available-systems.md#lumi)
+is one of the fastest in the world. It is intended primarily
+for running computations that benefit from the large amount of high-performance
+GPUs in its LUMI-G hardware partition. Whereas the GPUs on Puhti and Mahti are
+manufactured by Nvidia, the LUMI GPUs are made by AMD, so make sure that your
+GPU applications are able to run on AMD GPUs. LUMI has
+[its own documentation pages](https://docs.lumi-supercomputer.eu/).
+
```

Comment:
```suggestion

!!! note "On supercomputing"
    CSC supercomputers offer resources that, when properly used, are well-beyond
    what the most sophisticated consumer devices are capable of. However, *you are
    not the only one using them*. On your personal workstation, you have, in
    principle, immediate access to resources. On a supercomputer, which is a shared
    system, you must typically queue for them, since their demand tends to be higher
    than their supply. See our [usage policy](../../computing/usage-policy.md)
    for more information.

    It is also worth keeping in mind that running your computations on a
    supercomputer only improves performance if you play to its strengths.
    Supercomputers are powerful because they allow for
    [parallel computing](https://en.wikipedia.org/wiki/Parallel_computing).
    If your code is not written to take advantage of multiple CPU cores, or one or
    more GPUs, there might be no benefit over running it on your own workstation.
    However, high memory and/or storage requirements, as well as availability of pre-installed
    software and licenses are other factors which may make using CSC supercomputers
    attractive for you.

```